stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/adapter/viewholder/decorator/TimeDecorator.kt

```diff
@@ -0,0 +1,35 @@
+package io.getstream.chat.android.ui.messages.adapter.viewholder.decorator
+
+import android.widget.TextView
+import com.getstream.sdk.chat.adapter.MessageListItem
+import com.getstream.sdk.chat.utils.DateFormatter
+import com.getstream.sdk.chat.utils.formatTime
+import io.getstream.chat.android.ui.messages.adapter.viewholder.MessagePlainTextViewHolder
+import io.getstream.chat.android.ui.messages.adapter.viewholder.OnlyMediaAttachmentsViewHolder
+import io.getstream.chat.android.ui.messages.adapter.viewholder.PlainTextWithMediaAttachmentsViewHolder
+import io.getstream.chat.android.ui.utils.extensions.getCreatedAtOrThrow
+
+internal class TimeDecorator(private val dateFormatter: DateFormatter) : BaseDecorator() {
+
+    override fun decoratePlainTextMessage(viewHolder: MessagePlainTextViewHolder, data: MessageListItem.MessageItem) {
+        setupTime(viewHolder.binding.tvTime, data)
+    }
+
+    override fun decoratePlainTextWithMediaAttachmentsMessage(
+        viewHolder: PlainTextWithMediaAttachmentsViewHolder,
+        data: MessageListItem.MessageItem
+    ) {
+        setupTime(viewHolder.binding.tvTime, data)
```

Comment:
I thought it should show time only for messages with the bottom position, not every message. The same about avatar. Or it does not suppose to be part of this PR?